helm.go

```diff
@@ -185,8 +185,12 @@ This will not alter the charts in your workspace.
 			ArgsUsage: "[chart-name...]",
 			Action: func(c *cli.Context) {
 				minArgs(c, 1, "uninstall")
+
+				// Todo: add dry-run flag
+				client := kubectl.Client
```

Comment:
No, I should update my Todo statement to read 'Create a ticket to add `dry-run` flag to uninstall'